File: proj/CoreLibrary/src/com/jasonzqshen/familyaccounting/core/exception/NoFieldNameException.java
package com.jasonzqshen.familyaccounting.core.exception;

public class NoFieldNameException extends Exception {

	/**
	 * 
	 */
	private static final long serialVersionUID = -8358659959308638537L;

	public NoFieldNameException(String fieldName) {
		super("No such field " + fieldName);
	}
}

File: trunk/proj/WinStore_FamilyAccounting/WinStore_FamilyAccountingCore/Exceptions/NoFieldNameException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WinStore_FamilyAccountingCore.Exceptions
{
    public class NoFieldNameException : Exception
    {
        public NoFieldNameException(String fieldName)
            : base("No such field " + fieldName)
        {
        }
    }
}
